packages/toc/README.md

```diff
@@ -0,0 +1,52 @@
+# jupyterlab-toc
```

Comment:
The relevant contents of this file should be moved to the docs and then remove this file